src/libraries/System.Reflection.Emit/src/Resources/Strings.resx

```diff
@@ -297,4 +297,10 @@
   <data name="NotSupported_SymbolMethod" xml:space="preserve">
     <value>Not supported in an array method of a type definition that is not complete.</value>
   </data>
+  <data name="InvalidOperation_InvalidDocument" xml:space="preserve">
+    <value>Invalid source document.</value>
+  </data>
+  <data name="InvalidOperation_NotValidTypeBuilder" xml:space="preserve">
+    <value>Emitting symbols is not supported for containing type.</value>
```

Comment:
Could this message say why?

Reply:
This message will be thrown if the containing type of the local created by an `AssemblyBuilder` implementation that did override the `LocalBuilder.SetLocalSymInfoCore(string)` - that basically mean the implementation does not support PDB. The message is not perfect, but not sure how to improve, open for suggestions :)

Reply:
After thinking about it more, the code path that throws with this message is not expected to hit, kind of using is clause instead of casting `Type` into `TypeBuilder` which is not expected to fail, ended up removing this message.